lib/ModelMixins/FeatureInfoUrlTemplateMixin.ts

```diff
@@ -104,7 +104,22 @@ function FeatureInfoUrlTemplateMixin<T extends AbstractConstructor<BaseType>>(
 
     wrapImageryPickFeatures<T extends ImageryProvider>(imageryProvider: T) {
       const realPickFeatures = imageryProvider.pickFeatures;
-      const catalogItem = this;
+
+      // Get CatalogItem properties, so we aren't reading them outside of a reaction
+      const featureInfoUrlTemplate = this.featureInfoUrlTemplate;
+      const proxiedFeatureInfoUrl = proxyCatalogItemUrl(
+        this,
+        this.featureInfoUrlTemplate ?? "",
+        "0d"
+      );
+      const showStringIfPropertyValueIsNull = MappableMixin.isMixedInto(this)
+        ? this.showStringIfPropertyValueIsNull
+        : undefined;
+      const maxRequests = this.maxRequests;
+      const currentTime =
+        (TimeVarying.is(this) ? this.currentTimeAsJulianDate : undefined) ??
+        JulianDate.now();
+
```

Comment:
This fixes the "read outside reactive context" warnings we were getting with `featureInfoUrlTemplate`